core/domain/opportunity_services.py

```diff
@@ -107,18 +107,21 @@ def get_exploration_opportunity_summary_from_model(
         {})
 
 
-def _save_multi_exploration_opportunity_summary(
+def _create_opportunity_summary_models(
```

Comment:
"create" currently has semantics of save. Either we change to create for no-save and create-and-put for save throughout, or this should be renamed to _construct_new_opportunity_summary_models() (or similar).